prepare/metrics/kendalltau.py

```diff
@@ -21,6 +21,31 @@
     metric=KendallTauMetric(),
 )
 
+
+singleton_instance_target = [
```

Comment:
do you think we need single instance test also for spearmanr?

Reply:
After the recent change that we discussed, we don't need it for either of them.